src/librustdoc/passes/collect_trait_impls.rs

```diff
@@ -67,16 +67,14 @@ pub fn collect_trait_impls(krate: Crate, cx: &DocContext<'_>) -> Crate {
         if !def_id.is_local() {
             inline::build_impl(cx, def_id, &mut new_items);
 
-            let auto_impls = get_auto_traits_with_def_id(cx, def_id);
-            let blanket_impls = get_blanket_impls_with_def_id(cx, def_id);
-            let mut renderinfo = cx.renderinfo.borrow_mut();
+            // FIXME(eddyb) is this `doc(hidden)` check needed?
```

Comment:
Well, looks like yes in theory, but not in practice because none of the "primitive" impls is hidden.

Reply:
cc @QuietMisdreavus